assets/alignmentchart-style.css

```diff
@@ -107,49 +59,32 @@
     width: 50% !important;
     margin-left: 25px;
 }
-.alignment-settings > div {
-    display: inline-block;
-    width: 65%
-}
-.alignment-setting-name {
-    margin-top: 10px;
-    margin-bottom: 10px;
-    margin-left: 10px;
-    font-size: 15pt;
-    display: inline-block !important;
-    width: 30% !important;
-    vertical-align: middle;
-}
-
-.alignment-settings-dropdown, .alignment-radio {
-    margin-top: 10px;
-    margin-bottom: 10px;
-    width: 85% !important;
-    margin-left: 10%;
-    display: inline-block;
-    vertical-align: middle;
-}
-
-.alignment-settings p za{
-    margin-top: 5px;
-    margin-left: 15px;
-    margin-bottom: 20px;
-}
 
 .alignment-radio > label {
-    width: 45%;
     text-align: left;
     display: block;
 }
 .alignment-radio {
-    width: 50% !important;
-    margin-left: 20px !important;
+    margin-top: 5px;
+    margin-left: 5px;
+    width: 140px !important;
+    display: inline-block;
 }
 h3.alignment-settings-section {
     text-align: center;
-    margin-bottom: 20px;
+    margin-bottom: 15px;
+    font-size: 16pt;
 }
 
-h4 {
-    text-align: center;
+#alignment-colorscale-dropdown, #alignment-overview-dropdown, #alignment-conservationcolorscale-dropdown, #alignment-conservationmethod-dropdown, #alignment-gapcolor-dropdown {
```

Comment:
Is it possible to break this line? I'm not familiar enough with CSS syntax to make a suggestion.

Reply:
Done in e492ed6!

Reply:
@shammamah I guess you meant b6388d4? It looks like e492ed6 shouldn't be in (it's an empty commit). Do you mind removing it by rebasing interactively?